### PR TITLE
WIP: Codebase cleanup

### PR DESCRIPTION
**Pull request by btenterprise2020**
resolves https://github.com/ethereumclassic/explorer/issues/278
- [x] Merged master branch
- [x] Updated LICENSE to be clear
- [x] Updated Font Awesome to latest version
- [x] Removed broken WATCHING button
- [x] Fixed uncle 404 error
- [x] Enforce lowercase on transactions & accounts when recording on DB

### CODE REVIEW

**Comment by btenterprise2020** [comment]
Ready to merge.

**Comment by hackmod** [comment]
please do not mix separatable PRs in one PR.

**Comment by btenterprise2020** [comment]
> please do not mix separatable PRs in one PR.

I will add more features on this PR thanks.

**Comment by btenterprise2020** [comment]
> please do not mix separatable PRs in one PR.
@hackmod Are you in charge of this explorer?

File: config.example.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "nodeAddr":     "localhost",
-    "gethPort":     8545,
+    "rpcPort":      8545,
     "startBlock":   0,
     "quiet":        true,
     "syncAll":      true,
```
Comment (tdmllc1) [inline config.example.json]: 500000000000000 quiet tdmllc1@outlook.com liquidity for binance BTCUDST

**Pull request closed without merging**